.github/workflows/push-release.yml

```diff
@@ -19,11 +19,3 @@ jobs:
           name: ${{ env.DOCKERHUB_REPOSITORY }}
           username: ${{ env.DOCKERHUB_USERNAME }}
           password: ${{ env.DOCKERHUB_PASSWORD }}
-          tags: 'latest'
```

Comment:
tags: 'latest' has to be removed here, so that DockerHub build tagged versions based on github branch name